CHANGELOG.md

```diff
@@ -1,3 +1,103 @@
+### citus v9.5.0 (November 10, 2020) ###
+
+* Adds support for PostgreSQL 13
+
+* Removes the task-tracker executor
+
+* Introduces citus local tables
+
+* Introduces `undistribute_table` UDF to convert tables back to postgres tables
+
+* Adds support for `EXPLAIN ANALYZE EXECUTE` and `EXPLAIN EXECUTE`
+
+* Adds support for `EXPLAIN(ANALYZE, WAL)` for PG13
+
+* Adds support for `WITH TIES` option in SELECT and INSERT SELECT queries
+
+* Avoids taking multi-shard locks on workers
+
+* Enforces `citus.max_shared_pool_size` config in COPY queries
+
+* Enables custom aggregates with multiple parameters to be executed on workers
+
+* Enforces `citus.max_intermediate_result_size` in local execution
+
+* Improves cost estimation of INSERT SELECT plans
+
+* Introduces delegation of procedures that read from reference tables
+
+* Prevents pull-push execution for simple pushdownable subqueries
+
+* Sorts the output of `EXPLAIN ANALYZE` by execution duration.
```

Comment:
I would put this together with the other EXPLAIN ANALYZE improvements.

Reply:
makes sense

Reply:
done 0f5c6d276e0d5c3d0ad562ecb9f9bbab6e946e0b